fe/app/components/BannerGrid/BannerGrid.tsx

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import ImageElement from "../ImageElement/ImageElement";
+import { BannerGridProps } from "./BannerGrid.models";
+
+const BannerGrid = ({ images }: BannerGridProps) => {
+  return (
+    <div>
```

Comment:
Since it has no properties, it really has to be a **_div_** or could be an empty tag **<>**?